Tools/Scripts/webkitpy/w3c/test_downloader.py

```diff
@@ -93,6 +94,8 @@ def checkout_test_repository(self, revision, url, directory):
             git = self.git(directory)
         _log.info('Checking out revision ' + revision)
         git.checkout(revision, not self._options.verbose)
+        # to be used when logging in test_importer for putting inside a PR
+        self.upstream_commitID_head = git.rev_parse('HEAD')
```

Comment:
By only initialising this here, we only have this in the case when `-s` is not passed. Maybe that's good, given the local working copy could have changes in it, but it is perhaps surprising for people who use their own pre-existing WPT clone?

Reply:
Let's try something else.